### PR TITLE
some message

### PR DESCRIPTION
**Pull request by rekhakilari**
- [DEMO LINK](https://rekhakilari.github.io/layout_product-cards-en/)
- [TEST REPORT LINK](https://rekhakilari.github.io/layout_product-cards-en/report/html_report/)

### CODE REVIEW

**Comment by rekhakilari** [comment]
- [DEMO LINK](https://rekhakilari.github.io/layout_product-cards-en/)
- [TEST REPORT LINK](https://rekhakilari.github.io/layout_product-cards-en/report/html_report/)

**Review by rHashTag — CHANGES_REQUESTED** [review]
Great work on the code! Looks amazing!

Some feedback:

Try not to commit unnecessary files such as package-lock.json and package.json.

You can overcome this by typing the following in terminal instead of `git commit -am "message"`:

`git add src` or `git add src/` or `git add src/index.html src/style.css`

then

`git commit -m "message"`

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  background-color: #00acdc;
```
Comment (rHashTag) [inline src/style.css:106]: A suggestion, is perhaps creating a variable for the color you use more than once, like `#00acdc`

**Review by rHashTag — CHANGES_REQUESTED** [review]
Good job! Here are some changes I'd suggest.

**Comment by rekhakilari** [comment]
- [DEMO LINK](https://rekhakilari.github.io/layout_product-cards-en/)
- [TEST REPORT LINK](https://rekhakilari.github.io/layout_product-cards-en/report/html_report/)

**Comment by rekhakilari** [comment]
- [DEMO LINK](https://rekhakilari.github.io/layout_product-cards-en/)
- [TEST REPORT LINK](https://rekhakilari.github.io/layout_product-cards-en/report/html_report/)

**Pull request closed without merging**